src/server/scripts/Kalimdor/TempleOfAhnQiraj/instance_temple_of_ahnqiraj.cpp

```diff
@@ -67,15 +67,13 @@ class instance_temple_of_ahnqiraj : public InstanceMapScript
                 LoadDoorData(doorData);
                 IsBossDied[0] = false;
                 IsBossDied[1] = false;
-                IsBossDied[2] = false;
 
                 BugTrioDeathCount = 0;
 
                 CthunPhase = 0;
             }
 
-            //If Vem is dead...
-            bool IsBossDied[3];
+            bool IsBossDied[2];
```

Comment:
rename this to something more meaningful, Boss in the context of instance is a bit too generic imo